styles.css

```diff
@@ -42,25 +43,21 @@ header {
 #vierkantC {
     background-color: indianred;
     grid-area: vierkantC;
-    text-align: center;
 }
 
 #vierkantD {
     background-color: indianred;
     grid-area: vierkantD;
-    text-align: center;
 }
 
```

Comment:
Ik zou persoonlijk class en id namen geen camelcasing geven. In plaats daarvan zou ik een streepje gebruiken zoals vierkant-d als voorbeeld, dit is ook conventie in css. Het klopt dat in sommige programmeer talen zoals Java wel camelcase wordt gebruikt.